grails-app/controllers/com/unifina/controller/security/AuthController.groovy

```diff
@@ -70,7 +70,8 @@ class AuthController {
 		}
 
 		try {
-			user = userService.createUser([:] << cmd.properties << [username: invite.username])
+			def props = [:] << cmd.properties.findAll{ it != "dateCreated" } << [username: invite.username, lastLogin: new Date(0)]
```

Comment:
New users should get current time instead of `new Date(0)` as `lastLogin`. You can remove it from here if you put the default field value in the class.

Is the check for `dateCreated` needed? `RegisterCommand` doesn't have this property, so `it != "dateCreated"` should always be true?